extension/chrome/elements/pgp_block.ts

```diff
@@ -484,6 +494,13 @@ Catch.try(async () => {
     }
   };
 
+  const createEncryptedSubject = (subject: string) => {
+    return `<div style="font-size: 14px;"> Encrypted Subject:
+              <b> ${subject}</b>
+            </div>
+            <hr/>`;
```

Comment:
Please give the `div` a `border-bottom: 1px cacaca` (or similar color) instead. The hr looks ugly without further formatting.

(also, since it's a tiny bit of html, I'd put it on one line)

Reply:
I tried border bottom but it didn't work because we call `await renderContent(message.html(), false);`  and it sanitizes  HTML and removes the border. 
I think I can add border somehow but it will complicate the codebase

Reply:
I think I can add `encryptedSubject` parameter to this function 
```TS 
const separateQuotedContentAndRenderText = async (decryptedContent: string, isHtml: boolean)
```
 and in the end just prepend to the rendered message.
I think it won't complicate the codebase a lot. What do you think?

Reply:
Got it. We can, for now, leave the `<hr>` but I have to look into the sanitization. It should not be stripping away css styles. I'll make an issue for myself to look at it.

Reply:
> I think I can add `encryptedSubject` parameter to this function
> 
> ```ts
> const separateQuotedContentAndRenderText = async (decryptedContent: string, isHtml: boolean)
> ```
> 
> and in the end just prepend to the rendered message.
> I think it won't complicate the codebase a lot. What do you think?

It doesn't really matter where you put it as long as it's minimal and easy to follow.

Any code that is minimal and easy to follow is good code.

Go ahead & try it & evaluate it & see.

Reply:
Okay